src/schema/objects/metadata.yaml

```diff
@@ -2623,13 +2692,15 @@ _CoordUnits:
   - n/a
 _EEGCoordSys:
   type: string
+  # TODO: Add definitions for these values.
```

Comment:
What do the different `_EEGCoordSys` values mean?

Reply:
These are string identifiers that point to a free-form definition of

1. the origin of a coordinate system, defined in some meaningful way (e.g., by making cardinal axes go through well defined anatomical landmarks on a subject's head)
2. where the cardinal axes point to (e.g., y axis from origin (0) through nasion (positive))
3. the units for x,y,z values in this coordinate system

Reply:
I guess this is related to: https://github.com/bids-standard/bids-specification/issues/658